kuma/wiki/tests/test_kumascript.py

```diff
@@ -60,3 +81,166 @@ def test_get_does_not_call_format_slug_for_docs(self, mock_format_slug):
         kumascript.get(doc, 'no-cache', 'https://testserver')
         ok_(not mock_format_slug.called,
             "format slug should not have been called")
+
+
+@requests_mock.mock()
+def test_macro_sources(mock_requests):
+    """When KumaScript returns macros, the sources are populated."""
+    macros_url = urljoin(KUMASCRIPT_BASE_URL, 'macros/')
+    response = {
+        'can_list_macros': True,
+        'loader': 'FileLoader',
+        'macros': [{
+            'filename': 'A11yRoleQuicklinks.ejs', 'name': 'A11yRoleQuicklinks'
+        }, {
+            'filename': 'APIFeatureList.ejs', 'name': 'APIFeatureList'
+        }]
+    }
+    mock_requests.get(macros_url, json=response)
+    macros = kumascript.macro_sources()
+    expected = {
+        'A11yRoleQuicklinks': 'A11yRoleQuicklinks.ejs',
+        'APIFeatureList': 'APIFeatureList.ejs',
+    }
+    assert macros == expected
```

Comment:
@safwanrahman We shouldn't use the ``eq_``, ``ok_``, etc. helpers any longer, as pytest takes care of providing detailed information on failure using the standard ``assert``.